configuration/concepts/patient_outcome.csv

```diff
@@ -2,4 +2,5 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:f
 eb2cec8a-ee2e-4b12-9026-3d24e88bfab8,,,Referred By,Référencé par,Referred By,Référencé par,,,Misc,Text,,,,,,,,,,,
 68224cb3-5b61-410d-b269-7b4edc572fd8,,,Referral Notes,Notes de référence,Referral Notes,Notes de référence,,,Misc,Text,,,,,,,,,,,
 2ffc2e24-fa5c-4cb1-9366-8faec8225ee6,,,Left without permission,Parti sans autorisation,Left without permission,Parti sans autorisation,,,Misc,Coded,,Haiti_Yes;Haiti_No,,,,,,,,,
-16430ab8-5e6d-4031-a622-815fd0c9e41a,,,Comments,Commentaires,Comments,Commentaires,,,Misc,Text,,,,,,,,,,,
\ No newline at end of file
+16430ab8-5e6d-4031-a622-815fd0c9e41a,,,Comments,Commentaires,Comments,Commentaires,,,Misc,Text,,,,,,,,,,,
+bda6b247-f1fa-40d8-9164-79aebe99bfaf,,,Referral,Référencé,Referral,Référencé,,,ConvSet,N/A,,,Referred By; Health Centre; Other Health Centre; Referral Notes,,,,,,,,
```

Comment:
The term "Référencé" is no ideal here.
We are still to find the exact word in french here. I would suggest "Renvoi", which has multiple meanings but one of which is to refer to something or someone.

We should ask Catherine about this.

Reply:
Let's use "Référé"

Reply:
Let's use "Référence"

Reply:
I remove the accent on the last letter?

Reply:
Yes indeed. "Référencé" is equivalent to "referred" (as an adjective - participe passé in french).
Eg, "Referred patient" = "Patient référencé"

But "Référence" is the noun, equal to "Referral"